pkg/controllers/nodepool/hash/suite_test.go

```diff
@@ -63,13 +65,15 @@ var _ = Describe("Static Drift Hash", func() {
 	var nodePool *v1beta1.NodePool
 	BeforeEach(func() {
 		nodePool = test.NodePool(v1beta1.NodePool{
+			ObjectMeta: metav1.ObjectMeta{
+				Annotations: map[string]string{
+					v1beta1.NodePoolHashAnnotationKey:        "abceduefed",
```

Comment:
Should we add this in the generic case or just in the cases where we know that we want to validate that the hash annotation and the hash version are already on the NodePool

Reply:
Added the annotation based on each test case